charts/fluent-bit/values.yaml

```diff
@@ -0,0 +1,98 @@
+---
+version: v1  # service version, good practice to label every k8s resource with a version if we want to use istio traffic routing capabilities in the future
+
+nameOverride: "fluent-bit"  # replace this by your service name
+fullnameOverride: ""  # fill if you whish to override the generatedChartFullName
+name: &nameOverride
+
+# Controls if prometheus scrapping should be enabled for this service
+prometheus:
+  enabled: true  # enables/disable scraping of prometheus metrics
+  path: "/api/v1/metrics/prometheus"  # path for scrapping prometheus metrics
+  port: 2020  # port were the metrics web server is running
+  scheme: "tcp"  # tcp connection scheme
+
+# Service serviceAccount
+serviceAccount:
+  create: true  # if true the chart will create the service account
+  name: fluent-bit  # name of the serviceAccount to use or to create
+  annotations: {}
+    # eks.amazonaws.com/role-arn: arn:aws:iam::AWS_ACCOUNT_ID:role/IAM_ROLE_NAME
+
+# controls k8s Role Based Access Control settings tied to the serviceAccount
+rbac:
+  enabled: true  # enables/disables the creation of Role Based Policies
+  type: cluster  # type can be cluster or namespace  ( ClusterRole or Role)
+
+# pod affinity policies
+affinity: {}
+
+# node selector
+nodeSelector: {}
+
+# tolerations
+tolerations: []
+
+# service image
+image:
+  registry: docker.io  # container registry url
+  repository: amazon/aws-for-fluent-bit  # container registry repo name
+  tag: 2.3.0  # image tag
+  pullPolicy: Always  # kubernetes container image pull policy
+
+# limits the container resources
+resources:
+  # resources that the container requires to boot
+  requests:
```

Comment:
are you sure that it need this much?

Reply:
Saw that values in some random config. I will review the official config and copy the values from there.